### PR TITLE
Create new meta-information section and move existing rules

### LINKED ISSUE #349: Create meta information section and move existing rules
Move:

* [General Metadata (101)](http://zalando.github.io/restful-api-guidelines/#101),
* [API Identifier (215)](http://zalando.github.io/restful-api-guidelines/#215), and
* [Semantic Versioning (116)](http://zalando.github.io/restful-api-guidelines/#116)

to new metadata section to prepare additional metadata entries for

* API Audience (#348) and
* Callback APIs

in on place.

### PR DESCRIPTION
**Pull request by tkrop**
Fixes #349.

### CODE REVIEW

File: chapters/meta-information.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
+[[metadata]]
+= Meta Information
+
+== {MUST} Contain API Meta Information
```
Comment (ePaul) [inline chapters/meta-information.adoc]: Should this not have a rule number?

Comment (tkrop) [reply chapters/meta-information.adoc]: Good catch :)

**Comment by maxim-tschumak** [comment]
1. Please add information about the new chapter to the changelog.
2. Maybe it makes sense to move some guidance from 101 into the new chapter? (mandatory fields)

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Pull request merged**